tests/server/blueprints/variants/test_variants_views.py

```diff
@@ -96,8 +104,16 @@ def test_bulk_reset_dismiss_variants(app, institute_obj, case_obj):
         # GIVEN that the user could be logged in
         resp = client.get(url_for("auto_login"))
 
+        # GIVEN a variant
         variant = store.variant_collection.find_one()
 
+        # GIVEN a referal from a variants view
+        referer = url_for(
```

Comment:
I've also found this problem quite often and I've created a fixture to fix it, look above in the code

Reply:
Good, I didn't see it: feel free to change! I just followed your earlier example. 😁